scalahost/nsc/src/main/scala/scala/meta/internal/semantic/AttributesOps.scala

```diff
@@ -264,13 +265,15 @@ trait AttributesOps { self: DatabaseOps =>
               gtree match {
                 case gview: g.ApplyImplicitView =>
                   val pos = gtree.pos.toMeta
-                  val syntax = gview.fun + "(*)"
+                  val syntax = g.showCode(gview.fun) + "(*)"
```

Comment:
Here's an example failing test from rebase on master

```
[info]   -[273..275) h.X.cvt[Int](*)(h.C.int)
[info]   +[273..275) h.this.X.cvt[Int](*)(h.this.C.int)
```
The second one (.toString) would not compile if inserted into a source file while the first one (showCode) would compile.